### PR TITLE
0.11.0 change kv to label

### PR DESCRIPTION
**Pull request by domgreen**
Change `kv` to `label` as this has been updated in OTel `0.11.0`.

Release Notes https://github.com/open-telemetry/opentelemetry-go/releases/tag/v0.11.0

See https://github.com/open-telemetry/opentelemetry-go/pull/1060

Side Note:
codes should also be updated but would probably better do it in a different PR.

### CODE REVIEW

**Comment by domgreen** [comment]
I did look at updating code to the otel version but then realised that `spanData.StatusCode` does not use the new otel codes package. It currently still uses the grpc package.

**Comment by domgreen** [comment]
@paulosman @lizthegrey couldn't seem to assign reviewers, would you mind taking a look.

Have tested my fork against Honeycomb and all seems to be injesting fine from a client perspective.

File: example/client/client.go [hunk 1]
```diff
@@ -38,52 +40,56 @@ func main() {
 	dataset := flag.String("dataset", "opentelemetry", "Your Honeycomb dataset")
 	flag.Parse()
 
-	exporter, err := honeycomb.NewExporter(
+	exporter, errExp := honeycomb.NewExporter(
```
Comment (paulosman) [inline example/client/client.go:43]: Is there a reason to change this? If not, I'd prefer to leave it as it was, naming this `err` and using `err = func(ctx context.Context) error {}`  below.

**Review by paulosman — APPROVED** [review]
One small non-blocking comment. LGTM! Thanks so much for tackling this.

File: example/client/client.go (hunk 1 above)
Comment (domgreen) [reply example/client/client.go:43]: Ah, will fix that ... Just habit so err doesn't get shadowed.

Comment (domgreen) [reply example/client/client.go]: revert it back ... it is only example code :)

**Comment by domgreen** [comment]
@paulosman im not sure if any other updates are needed for it to be 0.11.0 compatible and tagged, I did update the version in preperation.
All `go.mod` are pointing at otel 0.11.0 ... if you could double check to see if I missed anything and then maybe tag `0.11.0`?

**Pull request merged**